playbooks/webapp/webapp_ha_aurora.yaml

```diff
@@ -1,106 +1,86 @@
 ---
-- name: Webapp HA
+- name: Configure inventory for High availability Aurora cluster
   hosts: localhost
   gather_facts: false
+
+  vars_files:
+    - vars/main.yaml
+
+  tasks:
+    - name: Add different hosts
+      ansible.builtin.add_host:
+        groups:
+          - aurora
+        name: "aurora_{{ item.region }}"
+        ansible_connection: local
+        region: "{{ item.region }}"
+        ansible_python_interpreter: "{{ ansible_python_interpreter }}"
+        deploy_flask_app_bastion_ssh_private_key_path: "~/.{{ resource_prefix }}{{ item.region }}_id_rsa"
+        rds_cluster_name: "{{ item.rds_cluster_name }}"
+      with_items:
+        - region: "{{ rds_primary_cluster_region }}"
+          rds_cluster_name: "{{ rds_primary_cluster_instance_name }}"
+        - region: "{{ rds_replica_cluster_region }}"
+          rds_cluster_name: "{{ rds_replica_cluster_instance_name }}"
+
+- name: Webapp HA
+  hosts: aurora
+  gather_facts: false
+  strategy: free
```

Comment:
I believe this prevents tasks from prematurely aborting on encountering a failure on one of the hosts. However, this may not be convenient when debugging the failure.